src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,24 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import './PostInfo.scss';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
```

Comment:
```suggestion
export const PostInfo = ({ post }) => {
  const {
    title,
    body,
    user,
    comments,
  } = post;
  
  return (
     <div className="PostInfo">
```
Use destructuring when you use a value from props many times